specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -769,6 +775,9 @@
           "items": {
             "$ref": "#/definitions/MetricCriteria"
           },
+          "x-ms-identifiers": [
+            "operator"
```

Comment:
A more appropriate identifier will probably be the "name" property.

Reply:
Thanks for looking at the code change, @asafst! I just pushed the fix regarding to your comments and please take a look when you get a chance.